### ISSUE TITLE
Only one  start and...

### ISSUE THREAD

**Issue by selevo** [Genymobile/scrcpy#2836]

**Comment by selevo** [comment]
win 7 x64
on start  after  download it's good connected.

All  next  start dont  work and  this  messages
`G:\Internet_download\Chrome\scrcpy-win64-v1.20>scrcpy.exe
INFO: scrcpy 1.20 <https://github.com/Genymobile/scrcpy>
* daemon not running; starting now at tcp:5037
adb F 11-29 14:43:26  9100  9184 main.cpp:52] cannot open C:\P\temp\adb.log: No such file or directory
could not read ok from ADB Server
* failed to start daemon
adb: error: failed to get feature set: cannot connect to daemon
ERROR: "adb push" returned with value 1
ERROR: Server connection failed`

And... about path C:\P\temp\
I wonder where scrcpy got this path from?
Because there is no such folder on my computer.
But in C:\P\  installed adroid studio

**Comment by rom1v** [comment]
> Only one start

Please test #2783.

> I wonder where scrcpy got this path from?

It's not scrcpy, it's adb. Do you use several instances of adb (one from Android Studio and one from scrcpy for example)?

**Comment by selevo** [comment]
> Do you use several instances of adb (one from Android Studio and one from scrcpy for example)?
No, i clicked  only scrcpy and  in  Task Manager no others adb.exe

**Comment by rom1v** [comment]
What is the result of `adb devices`?

**Comment by selevo** [comment]
> What is the result of `adb devices`?

G:\Internet_download\Chrome\scrcpy-win64-v1.20>^Vadb devices
"▬adb" не является внутренней или внешней
командой, исполняемой программой или пакетным файлом.

G:\Internet_download\Chrome\scrcpy-win64-v1.20>adb devices
* daemon not running; starting now at tcp:5037
adb F 11-29 16:21:37  5528  3680 main.cpp:52] cannot open C:\P\temp\adb.log: No such file or directory
could not read ok from ADB Server
* failed to start daemon
adb.exe: failed to check server version: cannot connect to daemon

G:\Internet_download\Chrome\scrcpy-win64-v1.20>

but  my brouser   detected  of  my phone
![2021-11-29~16_23_50](https://user-images.githubusercontent.com/13694921/143875989-ec74a347-4eb5-4ad3-8d90-fc6eb08008d0.gif)

**Comment by rom1v** [comment]
> Cannot open C:\P\temp\adb.log: No such file or directory

Probably your TMP/TEMP environment variable which is incorrect. See https://blog.csdn.net/QQ1452008/article/details/78871927 for example (it's in Chinese, but look at the screenshots).

**Comment by selevo** [comment]
> Probably your TMP/TEMP environment variable which is incorrect. 
Yes.. Thank's

The first thing I did was look at the environment variables.
And everything is standard there.

But now I looked again and upon close examination found that there was no colon sign, for some reason it disappeared.
It was 
TMP с \temp
TEMP c \temp

Has become
TMP с:\temp
TEMP c:\temp
1. There is also no Connection
2.  the error with a non-existent directory is gone.

After that, I rebooted the smartphone, turned off and then turned on ADB debugging, revoked the rights and played with other settings.
And after another crazy finger taps it all worked.

And webUSB  in brouser also worked
